src/manifolds/SymmetricPositiveDefiniteBuresWasserstein.jl

```diff
@@ -0,0 +1,137 @@
+@doc raw"""
+    BurresWassertseinMetric <: AbstractMetric
+
+The Bures Wasserstein metric for symmetric positive definite matrices[^MalagoMontruccioPistone2018].
+
+[^MalagoMontruccioPistone2018]:
+    > Malagò, L., Montrucchio, L., Pistone, G.:
+    > _Wasserstein Riemannian geometry of Gaussian densities_.
+    > Information Geometry, 1, pp. 137–179, 2018.
+    > doi: [10.1007/s41884-018-0014-4](https://doi.org/10.1007/s41884-018-0014-4)
+"""
+struct BuresWassersteinMetric <: RiemannianMetric end
+
+@doc raw"""
+    change_representer(M::MetricManifold{ℝ,SymmetricPositiveDefinite,BuresWassersteinMetric}, E::EuclideanMetric, p, X)
+
+Given a tangent vector ``X ∈ T_p\mathcal M`` representing a linear function on the tangent
+space at `p` with respect to the [`EuclideanMetric`](@ref) `g_E`,
+this is turned into the representer with respect to the (default) metric,
+the [`BuresWassersteinMetric`](@ref) on the [`SymmetricPositiveDefinite`](@ref) `M`.
+
+To be precise we are looking for ``Z∈T_p\mathcal P(n)`` such that for all ``Y∈T_p\mathcal P(n)```
+it holds
+
+```math
+⟨X,Y⟩ = \operatorname{tr}(XY) = ⟨Z,Y⟩_{\mathrm{BW}}
+```
+for all ``Y`` and hence we get
+``Z``= 2(A+A^{\mathrm{T}})`` with ``A=Xp``.
+"""
+change_representer(
+    ::MetricManifold{ℝ,SymmetricPositiveDefinite,BuresWassersteinMetric},
+    ::EuclideanMetric,
+    p,
+    X,
+)
+
+function change_representer!(
+    ::MetricManifold{ℝ,<:SymmetricPositiveDefinite,BuresWassersteinMetric},
+    Y,
+    ::EuclideanMetric,
+    p,
+    X,
+)
+    Y .= 2 .* (X * p + p' * X')
```

Comment:
```suggestion
    A = X * p
    Y .= 2 .* (A .+ A')
```
This should save one matrix multiplication.